pytensor/compile/builders.py

```diff
@@ -433,22 +377,25 @@ def __init__(
         self.grad_overrides = grad_overrides
```

Comment:
Does this need to be saved anymore? Should be unused as of this PR, to facilitate the shift to `lop_overrides` only

Reply:
It needs to be saved because the Op is sometimes reconstructed and there's special logic to wrap the user input to work with the L_op API that is only done lazily

Reply:
We'll simplify easy enough when we remove the deprecated kwarg